src/libraries/Common/tests/System/Security/Cryptography/AlgorithmImplementations/ECDsa/ECDsaTests.netcoreapp.cs

```diff
@@ -58,7 +58,8 @@ private static byte[] TryWithOutputArray(Func<byte[], (bool, int)> func)
     public abstract partial class ECDsaTests : ECDsaTestsBase
     {
         [Fact]
-        public void KeySizeProp()
+        [SkipOnMono("Android doesn't support getting a curve name from a curve.", TestPlatforms.Android)]
```

Comment:
Could you please annotate not supported curve name APIs with `UnsupportedOSPlatformAttribute("android")]`

Reply:
I'd rather mark the APIs when we get closer to finishing the Android crypto work in case we figure out a way to support these APIs instead of marking them now and then having to figure out which ones to un-mark later.

Reply:
Quick searching suggests that `PrivateKey::getEncoded()` returns the PKCS#8 blob.  That probably includes the encoded curve and further probably includes it as a named curve when the named curve is known.

Reply:
I found a way to do this on Android via an API exposed via reflection, so the API is no longer unsupported.